SteakBot.Core/EventHandlers/CustomMessageHandlers/CommandMessageHandlers/CustomCommandMessageHandler.cs

```diff
@@ -20,8 +20,21 @@ public CustomCommandMessageHandler(MemeService memeService)
         protected override bool InvokeInner(SocketUserMessage message)
         {
             var channel = message.Channel;
-            var commandText = message.Content.Replace(CommandChar, "").Replace(DeleteMessageChar, "");
-            var command = _memeService.MemeCommands.Single(x => x.Name == commandText);
+            var messageText = message.Content;
+
+            var startIndex = 0;
+            var endIndex = messageText.Length;
+            if (messageText.StartsWith(GlobalConstants.CommandChar))
+            {
+                startIndex++;
+            }
+            if (messageText.EndsWith(GlobalConstants.DeleteMessageChar))
+            {
+                endIndex--;
+            }
+            var commandName = messageText.Substring(startIndex, endIndex);
```

Comment:
This should be `Substring(startIndex, endIndex - startIndex + 1)`, since the method takes a `startIndex` and a `length`.

Reply:
Empty lines after blocks, please.